impeller/golden_tests/golden_playground_test_mac.cc

```diff
@@ -63,26 +63,15 @@ const std::unique_ptr<PlaygroundImpl>& GetSharedVulkanPlayground(
 // If you add a new playground test to the aiks unittests and you do not want it
 // to also be a golden test, then add the test name here.
 static const std::vector<std::string> kSkipTests = {
-    IMP_AIKSTEST(CanDrawPaintMultipleTimesInteractive),
-    IMP_AIKSTEST(CanRenderLinearGradientManyColorsUnevenStops),
-    IMP_AIKSTEST(CanRenderRadialGradient),
-    IMP_AIKSTEST(CanRenderRadialGradientManyColors),
     IMP_AIKSTEST(CanRenderBackdropBlurInteractive),
```

Comment:
The ones that are left use our imgui line drawing functionality which this doesn't catch.